src/components/SdkSection/SdkSection.module.css

```diff
@@ -0,0 +1,42 @@
+.sdkSection {
+    display: flex;
+    gap: 24px;
+
+    @media (max-width: 768px) { /* max-width: ${theme.layout.tablet} */
```

Comment:
Nit: Couldn't this be passed as css var?

Reply:
not unless I declare some css variable --layout-tablet: 768px somewhere. There are some variables already in the root, but I think they are specific to the doc, not the same as in ui-components theme. In any case there are other cases around the code where we use max-width: 768px like this. I used these css for styling because I thought it was the "standard" in the doc. Otherwise to use the variables I would have to rewrite it with styled components